src/cephadm/cephadm

```diff
@@ -1211,9 +1218,12 @@ def call(command,  # type: List[str]
     return out, err, returncode
 
 
-def call_throws(command, **kwargs):
-    # type: (List[str], Any) -> Tuple[str, str, int]
-    out, err, ret = call(command, **kwargs)
+def call_throws(command: List[str],
+         desc: Optional[str] = None,
+         verbosity: CallVerbosity = CallVerbosity.VERBOSE_ON_FAILURE,
+         timeout: Optional[int] = DEFAULT_TIMEOUT,
+         **kwargs) -> Tuple[str, str, int]:
+    out, err, ret = call(command, desc, verbosity, timeout, **kwargs)
```

Comment:
why not allow kwargs here?

Reply:
It still contains kwargs, which makes this hunk pretty much useless. As soon as you use kwargs, you pretty much loose the possibility to statically verify parameter usage altogether. Sigh. Any ideas? I mean, afaik we never make use of kwargs in the code base.